src/main/kotlin/no/nav/medlemskap/regler/funksjoner/RelasjonFunksjoner.kt

```diff
@@ -0,0 +1,64 @@
+package no.nav.medlemskap.regler.funksjoner
+
+import no.bekk.bekkopen.person.FodselsnummerValidator
+import no.nav.medlemskap.domene.*
+import no.nav.medlemskap.regler.common.Funksjoner
+import no.nav.medlemskap.regler.common.lagInterval
+import java.util.*
+
+object RelasjonFunksjoner {
+
+    fun List<Sivilstand>.hentFnrTilEktefellerEllerPartnerIPeriode(kontrollPeriode: Periode): List<String?> =
+            this.sivilstandSiste12Mnd(kontrollPeriode).filter {
+             it.type == Sivilstandstype.GIFT || it.type == Sivilstandstype.REGISTRERT_PARTNER}.map { it.relatertVedSivilstand }
+
+    fun List<Familierelasjon>.hentFnrTilBarnUnder25(): List<String?> =
+            this.filter {
+                         it.relatertPersonsRolle == Familierelasjonsrolle.BARN
+                        && FodselsnummerValidator.isValid(it.relatertPersonsIdent)
+                        && it.relatertPersonsIdent.filtrerBarnUnder25Aar()}
+                        .map { it.relatertPersonsIdent}
+
+    fun List<PersonhistorikkRelatertPerson>.hentRelatertSomFinnesITPS(relatert: List<String?>?):
+            List<PersonhistorikkRelatertPerson> =
+            this.filter { relatert?.contains(it.ident) ?: false && FodselsnummerValidator.isValid(it.ident)}}
+
+    fun String.filtrerBarnUnder25Aar() =
+            (Calendar.getInstance().get(Calendar.YEAR) - this.hentBursdagsAar().toInt()) <= 25
+
+    fun String.hentBursdagsAar(): String {
+      return this.hentAaHundre() + this.hent2DigitBursdagsAar()
+    }
+
+    fun String.hentAaHundre(): String? {
+      var resultat: String? = null
+      val individnummer: Int = getIndividnummer().toInt()
+      val birthYear: Int = hent2DigitBursdagsAar().toInt()
+
+        if (individnummer <= 499) {
+            resultat = "19"
+        } else if (individnummer >= 500 && birthYear < 40) {
+            resultat = "20"
+        } else if (individnummer >= 500 && individnummer <= 749 && birthYear >= 54) {
+            resultat = "18"
+        } else if (individnummer >= 900 && birthYear > 39) {
+            resultat = "19"
+        }
+          return resultat
+    }
+
```

Comment:
Pirk igjen, men if-else på denne måten er ikke "the kotlin way". Trenger ikke gjøre noe nå, men for eksempel dette:
```
        return when {
            individnummer <= 499 -> "19"
            individnummer >= 500 && birthYear < 40 -> "20"
            individnummer in 500..749 && birthYear >= 54 -> "18"
            individnummer >= 900 && birthYear > 39 -> "19"
            else -> null
        }
```
Synes jeg er mer lesbart. Da kan man også droppe resultat-variabelen.

Reply:
Har fikset dette.